wdl/gwas/regenie_sub.wdl

```diff
@@ -91,6 +91,19 @@ task step2 {
 
           for p in ${sep=" " phenolist}; do
 
+            N_cases_females=$(awk -v pheno=$p 'FNR==NR { females[$1]; next } FNR==1{ for(i=1;i<=NF;i++) {h[$i]=i} }; NR>1 && $h[pheno]==1 && $1 in females {print $1}' females <(zcat ${cov_pheno}) | wc -l)
+            N_cases_males=$(awk -v pheno=$p 'FNR==NR { males[$1]; next } FNR==1{ for(i=1;i<=NF;i++) {h[$i]=i} }; NR>1 && $h[pheno]==1 && $1 in males {print $1}' males <(zcat ${cov_pheno}) | wc -l)
+
+            echo "Female cases: "$N_cases_females
+            echo "Male cases: "$N_cases_males
+
+            if [[ $N_cases_females -lt 10 ]] || [[ $N_cases_males -lt 10 ]];
+            then
+              echo "Less than 10 cases in a sex. Skipping testing of sex specific effects."
+              touch ${prefix}"NOT_DONE.sex_spec.gz"
+              continue
+            fi
```

Comment:
Since outputs are globbed, touching a placeholder file is not necessary. IMO not having a file is more clear, but that is up to taste

Reply:
Especially since if there are no sex-specic files, gather step will touch a gathered regenie sex-specific file. Not sure how tabix will work with an empty file.

Reply:
Good point! Didn't actually even notice the touch in gather but was just following what was done at the end of step2:

https://github.com/FINNGEN/regenie-pipelines/blob/42e4e820cb14051c0d7716c4e05c1170132dddf3/wdl/gwas/regenie_sub.wdl#L208-L215